frontend/src/scenes/experiments/ExperimentView/components.tsx

```diff
@@ -232,3 +233,86 @@ export function PageHeaderCustom(): JSX.Element {
         />
     )
 }
+
+export function ActionBanner(): JSX.Element {
+    const {
+        experiment,
+        experimentResults,
+        experimentLoading,
+        experimentResultsLoading,
+        isExperimentRunning,
+        areResultsSignificant,
+        isExperimentStopped,
+    } = useValues(experimentLogic)
+
+    if (!experiment || experimentLoading || experimentResultsLoading) {
+        return <></>
+    }
+
+    // Draft
+    if (!isExperimentRunning) {
+        return (
+            <LemonBanner type="info" className="mt-4">
+                Your experiment is in draft mode. You can edit your variants, adjust release conditions, and test your{' '}
+                <Link className="font-semibold" to="https://posthog.com/docs/experiments/testing-and-launching">
+                    test your feature flag
+                </Link>
+                . Once everything works as expected, you can launch your experiment. From that point, any new experiment
+                events will be counted towards the results.
+            </LemonBanner>
+        )
+    }
+
+    // Running, results present, not significant
+    if (isExperimentRunning && experimentResults && !isExperimentStopped && !areResultsSignificant) {
```

Comment:
Hmm, is there any case where we want to give different advice based on why its not significant?

If less than 100 people seen, this makes sense.

But for loss/calc, probability less than 90%, it might not necessarily make sense to wait for more data - assuming the experiment has run sufficiently long. In this case, we can say something along the lines of  'cut your losses, it doesn't seem like this experiment will be significant based on the results we've seen so far'